.travis.yml

```diff
@@ -0,0 +1,21 @@
+dist: xenial
+# Use the latest Travis images since they are more up to date than the stable release.
+group: edge
+sudo: false
+jobs:
+    include:
+    
+    # Run JS tests
+    - language: node_js
+      # The Node version here must be kept in sync with that in `package.json`.
+      node_js:
+        - "14"
+        - "13"
+        - "12"
+     
+     script:
+        - npm run lint
+        - npm test
+        - npm build
+        - pre-commit run --all
```

Comment:
This won't work. Pre commit is python based. Create a python based matrix different. Refer to officialWebsite Travis file

Reply:
Okay